<?xml version="1.0" encoding="UTF-8"?>

<launch>
	<arg name="rob_1"/>
	<arg name="rob_2"/>
	<node pkg="real_robot_controller" type= "multi_global_planner_agentpos_pub.py" name = "multi_global_planner_agentpos_pub" output="screen" args="$(arg rob_1) $(arg rob_2)"/>
	<node pkg="real_robot_controller" type= "multi_global_planner_client.py" name = "multi_global_planner_client" output="screen"/>
	<node pkg="real_robot_controller" type= "multi_global_planner_server.py" name = "multi_global_planner_server" output="screen"/>
</launch>
